mmv1/third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -124,8 +124,14 @@ The following arguments are supported:
     field type, we currently cannot suppress the recurring diff this causes.
     As a workaround, we recommend using the schema as returned by the API.
 
-    ~>**NOTE:**  When setting `schema` for `external_data_configuration`, please use
-    `external_data_configuration.schema` [documented below](#nested_external_data_configuration).
+    ~>**NOTE:**  If you are working with `external_data_configuration`
```

Comment:
This is a little convoluted - I think it might be possible to clarify by including the full condition at the beginning of the sentence. We can also leave off the thing that will happen if they disregard the advice - it's not bad enough that it needs to be called out specifically.

What would you think about something like:

```suggestion
    ~>**NOTE:**  If you use `external_data_configuration` and do **not** set `external_data_configuration.connection_id`, schemas must be specified with `external_data_configuration.schema`. Otherwise, schemas must be specified with this top-level field.
```